packages/babel-plugin-material-ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@material-ui/babel-plugin-material-ui",
+  "version": "1.0.0",
+  "description": "A simple transform to cherry-pick Material-UI imports",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "author": "Material-UI Team",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui-org/material-ui.git",
+    "directory": "packages/babel-plugin-material-ui"
+  },
+  "bugs": {
+    "url": "https://github.com/mui-org/material-ui/issues"
+  },
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "test": "yarn build && cd ../../ && cross-env NODE_ENV=test mocha 'packages/babel-plugin-material-ui/test/index.js'"
+  },
+  "devDependencies": {
+    "@types/babel__core": "^7.1.2",
+    "babel-plugin-tester": "^7.0.1",
+    "typescript": "^3.5.3"
+  },
+  "dependencies": {
+    "@babel/core": "^7.5.5",
```

Comment:
I think plugins need to list the core as a peer dependencies. The plugin gets the types injected from the API argument i.e. move core to peer and don't require types (and remove it from deps).